OpenSim/Simulation/SimbodyEngine/Constraint.cpp

```diff
@@ -71,7 +71,29 @@ void Constraint::setNull(void)
  */
 void Constraint::constructProperties(void)
 {
-    constructProperty_isDisabled(false);
+    constructProperty_isEnforced(true);
+}
+
+void Constraint::updateFromXMLNode(SimTK::Xml::Element& node,
+                                   int versionNumber) {
+    if(versionNumber < XMLDocument::getLatestVersion()) {
+        if(versionNumber < 30508) {
+            // Rename property 'isDisabled' to 'isEnforced' and
+            // negate the contained value.
+            std::string oldName{"isDisabled"};
+            std::string newName{"isEnforced"};
+            if(node.hasElement(oldName)) {
+                auto elem = node.getRequiredElement(oldName);
+                elem.setElementTag(newName);
+                if(elem.getValue().find("true") != std::string::npos)
+                    elem.setValue("false");
+                else if(elem.getValue().find("false") != std::string::npos)
+                    elem.setValue("true");
```

Comment:
Should have an `else` clause that throws an exception if someone tries to set `isEnforced` to `giraffe`. Tracked in #686.